packages/communication-react/CHANGELOG.stable.md

```diff
@@ -7,104 +7,45 @@ This log was last generated on Thu, 14 Mar 2024 03:50:48 GMT and should not be m
 ## [1.14.0](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.14.0)
```

Comment:
The link needs to be changed to https://github.com/azure/communication-ui-library/tree/1.14.0

Reply:
done